### PR TITLE
LPS-132249 Test Autom: Tie vocabularies to session

### PR DESCRIPTION
**Pull request by Luizcarlosqueiroz**
**JIRA Ticket:**
LPS-132249
LPS-107362

**Motivation:**
Test Automation for "Tie vocabularies to existing session criteria"

### CODE REVIEW

**Comment by Luizcarlosqueiroz** [comment]
@CarlosBrichete

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardAuditGraph.testcase [hunk 1]
```diff
@@ -585,4 +585,100 @@ definition {
...
+			Segmentation.gotoSidebar(fieldType = "Session");
+
+			Segmentation.addProperties(propertyOption = "Browser");
+
+			Segmentation.fillProperty(optionsValue = "Audience 1");
+
+			Segmentation.fillTitle(titleField = "KeepInfomationOfDeleteVocabularyInSegment Test");
+
+			Button.clickSave();
+		}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardAuditGraph.testcase:623]: Can you create this segment using the JSON macro?

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/contentdashboard/ContentDashboard.path [hunk 2]
```diff
@@ -23,6 +23,16 @@
...
+	<td>AUDIT_GRAPH_CONFIGURATION_LEFT_ARROW_BUTTON</td>
+	<td>//button[contains(@class, 'move-left')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>AUDIT_GRAPH_CONFIGURATION_RIGHT_ARROW_BUTTON</td>
+	<td>//button[contains(@class, 'move-right')]</td>
+	<td></td>
+</tr>
 <tr>
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/contentdashboard/ContentDashboard.path:36]: Aren't this buttons already created in the Button.path? Can you use a generic button path?

Comment (Luizcarlosqueiroz) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/contentdashboard/ContentDashboard.path:36]: I couldn't find these button there, I could only find an arrow down. And the issue with a generic path is that they all require 'key_text', and those two buttons don't have any.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardAuditGraph.testcase (hunk 1 above)
[the remark this answers is not in the document]
Comment (Luizcarlosqueiroz) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardAuditGraph.testcase:623]: With I tinny change, I could use the macro. I'll send in next PR.

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
@Luizcarlosqueiroz tests are failing... please review :)

**Comment by Luizcarlosqueiroz** [comment]
ci:test:tango

**Comment by Luizcarlosqueiroz** [comment]
ci:test:tango

**Comment by Luizcarlosqueiroz** [comment]
ci:test:tango

**Comment by Luizcarlosqueiroz** [comment]
@CarlosBrichete I think it's fixed now.

**Comment by CarlosBrichete** [comment]
ci:forward

**Pull request closed without merging**